tests/test_skvbc_auto_view_change.py

```diff
@@ -51,15 +51,16 @@ class SkvbcAutoViewChangeTest(unittest.TestCase):
         """
         bft_network.start_all_replicas()
 
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+
         initial_primary = 0
 
         # do nothing - just wait for an automatic view change
-        await bft_network.wait_for_view_change(
-            replica_id=random.choice(
-                bft_network.all_replicas(without={initial_primary})),
-            expected=lambda v: v > initial_primary,
-            err_msg="Make sure automatic view change has occurred."
-        )
+        await bft_network.wait_for_view(replica_id=random.choice(
+            bft_network.all_replicas(without={initial_primary})), expected=lambda v: v > initial_primary,
+            err_msg="Make sure automatic view change has occurred.")
```

Comment:
Can you preserve the existing code formatting, and only change the function name?

Reply:
i have changed only the name, the format is changed because the function name has been shorter